### ISSUE TITLE
**FEATURE**Adding Linear fan speed control between the thresholds

### ISSUE THREAD

**Issue by Stavro-Purdie** [nmaggioni/r710-fan-controller#14]
Hey guys,
I have been running your software for a while now and it works flawlessly, but I couldn't help but wonder what it would be like if we added in linear fan control that operated between the 3 thresholds, or increase the threshold limits to that we can add more fan states.

**Comment by nmaggioni** [comment]
Good idea, even though it will need testing since I'm not sure that all series of BMCs support arbitrary fan speeds.
I designed the current behavior with minimal noise variations in mind: depending on your specific use case and ambient conditions, frequent fan speed changes may be more noticeable than sustained overall higher speeds.

Anyway, PRs are welcome!
